hub/powertoys/advanced-paste.md

```diff
@@ -1,14 +1,14 @@
 ---
 title: PowerToys Advanced Paste for Windows
-description: An AI enhanced tool that enables you to intelligently paste the text from your clipboard into any format needed.
+description: A tool that enables you to paste the text from your clipboard into any format needed. It can be enhanced with an AI powered option that is 100% opt-in and requires an Open AI key.
```

Comment:
```suggestion
description: A tool that enables you to paste the text from your clipboard into any format needed. It can be enhanced with an AI-powered option that is 100% opt-in and requires an OpenAI key.
```